lib/ex_force/request.ex

```diff
@@ -0,0 +1,24 @@
+defmodule ExForce.Request do
+  @moduledoc """
+  Represents an ExForce API request.
```

Comment:
Sounds like "ExForce API" request. Probably better to describe what it conceptually represents.

Like: 

> Represents an API request.

Reply:
Fixed with cdd516e792e1707d72a4cf1eb0e7961879069bdd